packages/@aws-cdk/aws-appmesh/test/test.virtual-gateway.ts

```diff
@@ -238,6 +238,48 @@ export = {
       test.done();
     },
   },
+
+  'When creating a VirtualGateway with backend defaults': {
+    'should add backend defaults to the VirtualGateway'(test: Test) {
+      // GIVEN
+      const stack = new cdk.Stack();
+
+      // WHEN
+      const mesh = new appmesh.Mesh(stack, 'mesh', {
+        meshName: 'test-mesh',
+      });
+
+      new appmesh.VirtualGateway(stack, 'virtual-gateway', {
+        virtualGatewayName: 'virtual-gateway',
+        mesh: mesh,
+        backendDefaultsClientPolicy: appmesh.ClientPolicy.fileTrust({
+          certificateChain: 'path-to-certificate',
+        }),
+      });
+
+      // THEN
+      expect(stack).to(haveResourceLike('AWS::AppMesh::VirtualGateway', {
+        VirtualGatewayName: 'virtual-gateway',
+        Spec: {
+          BackendDefaults: {
+            ClientPolicy: {
+              TLS: {
+                Validation: {
+                  Trust: {
+                    File: {
+                      CertificateChain: 'path-to-certificate',
+                    },
+                  },
+                },
+              },
+            },
+          },
+        },
+      }));
+      test.done();
```

Comment:
I usually like to leave an empty line before the last `test.done()`, makes the tests read slightly better in my view.